### ISSUE TITLE
Ensure form inputs have labels

### ISSUE THREAD

**Issue by nwalters512** [PrairieLearn/PrairieLearn#10487]
This is important for screen reader users.

Known issues from accessibility audits:

- Input elements for questions don't use `aria-label` or `aria-labelledby`. Some use `aria-describedby`, but this is insufficient to ensure that they have a label/name.
- Search fields on the "Questions" table.

**Comment by nwalters512** [comment]
Related: https://github.com/PrairieLearn/PrairieLearn/issues/7385
